src/main/Port.Adapter/UI/Views/Blazor/Shared/BlazoriseSidebar.razor

```diff
@@ -16,7 +17,7 @@
         <BarItem>
             <BarLink To="">
                 <BarIcon IconName="brandLogo" />
-                neurUL &lt;#
+                @this.title
```

Comment:
Please use ei8.Cortex.Diary.Application.Settings.ISettingsService similar to BasePath